src/cli.rs

```diff
@@ -88,6 +88,13 @@ pub(crate) fn build_cli() -> Command {
             .env("KUBEWARDEN_KEY_FILE")
             .help("Path to an X.509 private key file for HTTPS"),
 
+        Arg::new("client-ca-cert-file")
+            .long("client-ca-cert-file")
+            .value_name("CLIENT_CA_CERT_FILE")
+            .default_value("")
+            .env("KUBEWARDEN_CLIENT_CA_FILE")
+            .help("Path to an X.509 certificate file for mTLS"),
```

Comment:
Also, this help message is wrong. This is the CA that issued the client certificat